geomesa-core/src/main/scala/org/locationtech/geomesa/core/stats/StatWriter.scala

```diff
@@ -80,7 +80,9 @@ object StatWriter extends Runnable with Logging {
    * Starts the background thread for writing stats, if it hasn't already been started
    */
   private def startIfNeeded() {
+    logger.debug("Starting StatWriter outside If Statement")
     if (running.compareAndSet(false, true)) {
+      logger.debug("Starting StatWriter inside If Statement")
```

Comment:
These lines were helpful for debugging.  We can remove them now that things are working.